src/site/scripts/components/profile/actions.tsx

```diff
@@ -28,6 +33,7 @@ export interface IActionProps {
  * Component for actionable profile buttons.
  */
 export const Actions: React.StatelessComponent<IActionProps> = (props: IActionProps): JSX.Element => {
+    let killButton  = `I killed ${props.target}!`;
```

Comment:
Why declare this if it' s only used once?

Also, nit: prefer `const` over `let`.